src/webgpu/shader/validation/parse/statement_behavior.spec.ts

```diff
@@ -0,0 +1,143 @@
+export const description = `
+Test statement behavior analysis.
+
+Functions must have a behavior of {Return}, {Next}, or {Return, Next}.
+Functions with a return type must have a behavior of {Return}.
+
+Each statement in the function must be valid according to the table.
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const kInvalidStatements = {
+  break: `break`,
+  break_if: `break if true`,
+  continue: `continue`,
+  loop1: `loop { }`,
+  loop2: `loop { continuing { } }`,
+  loop3: `loop { continue; continuing { } }`,
+  loop4: `loop { continuing { break; } }`,
+  loop5: `loop { continuing { continue; } }`,
+  loop6: `loop { continuing { return; } }`,
+  loop7: `loop { continue; break; }`,
+  loop8: `loop { continuing { break if true; return; } }`,
+  for1: `for (;;) { }`,
+  for2: `for (var i = 0; ; i++) { }`,
+  for3: `for (;; break) { }`,
+  for4: `for (;; continue ) { }`,
+  for5: `for (;; return ) { }`,
+  for6: `for (;;) { continue; break; }`,
+  // while loops always have break in their behaviors.
+  switch1: `switch (1) { case 1 { } }`,
```

Comment:
This one is invalid because there's no default clause - was there something specific to the behavior analysis that it was supposed to be testing instead?